tests/addresses_test.js

```diff
@@ -13,24 +13,40 @@ const prism = new Prism(specFile, lobUri, process.env.LOB_API_TEST_TOKEN);
 
 // contract tests
 test("list addresses", async function (t) {
-  t.plan(1);
-  const response = await prism
-    .setup()
-    .then((client) =>
-      client.get(resource_endpoint, { headers: prism.authHeader })
-    );
-  t.assert(response.status === 200);
+  try {
+    const response = await prism
+      .setup()
+      .then((client) =>
+        client.get(resource_endpoint, { headers: prism.authHeader })
+      );
+    t.assert(response.status === 200);
+  } catch (prismError) {
+    if (Object.keys(prismError).length > 0) {
+      t.fail(JSON.stringify(prismError, null, 2));
+    } else {
+      t.fail(prismError.toString());
+    }
+  }
 });
 
 test("list addresses' parameters", async function (t) {
   const list = async (body) => {
-    const response = await prism.setup().then((client) =>
-      client.get(`${resource_endpoint}?${body}`, {
-        headers: prism.authHeader,
-      })
-    );
-    t.assert(response.status === 200);
-    return response.data;
+    try {
+      const response = await prism.setup().then((client) =>
+        client.get(`${resource_endpoint}?${body}`, {
+          headers: prism.authHeader,
+        })
+      );
+      t.assert(response.status === 200);
+      return response.data;
+    } catch (prismError) {
+      if (Object.keys(prismError).length > 0) {
+        t.fail(JSON.stringify(prismError, null, 2));
+      } else {
+        t.fail(prismError.toString());
+      }
+      return prismError;
```

Comment:
if the flow comes into this branch, it'd never make it to this `return` right? cuz the `t.fail` would stop the whole test?

Reply:
ah crap the return should be in the else block, thanks for catching that